src/containers/StudyCreateForm/StudyCreateForm.tsx

```diff
@@ -178,166 +150,296 @@ function StudyCreateFormContainer() {
+    const formData = new FormData();
+    formData.append('title', title);
+    formData.append('topic', topic);
+    formData.append('isOnline', isOnline.toString());
+    formData.append('region', isOnline ? '' : region);
+    formData.append('numberOfRecruits', recruitsNumber);
+    formData.append('startDateTime', startDate);
+    formData.append('endDateTime', endDate);
+    formData.append('preferredMBTIs', JSON.stringify(mbtiCheckedList));
+    formData.append('imageFile', imageSrc ? imageSrc : '');
+    formData.append('description', description);
+    return formData;
+  };
+
   return (
-    <form onSubmit={onSubmit}>
-      <StudyCreateWrapper>
-        <Typography variant='h4'>스터디 생성</Typography>
-        <LabelInput
-          id='study-title'
-          label='제목'
-          fullWidth={true}
-          onChange={(e) => {
-            setTitle(e.target.value);
-          }}
-        />
-        <Select
-          id='study-category'
-          label='분야'
-          fullWidth={true}
-          options={options}
-          getValue={getCategory}
-        />
-        <LocationWrapper>
-          <RadioGroup
-            defaultValue={radioValues[0].value}
-            labels={radioValues}
-            name='study-location'
-            row={true}
-            onChange={isOnlineChange}
-          />
-          <Select
-            id='study-location'
-            label='지역'
-            fullWidth={false}
-            options={locations}
-            getValue={getLocation}
-            disabled={isOnline ? true : false}
-          />
-        </LocationWrapper>
-        <Select
-          id='study-number-of-people'
-          label='인원'
-          fullWidth={true}
-          options={people}
-          getValue={getPeopleNumber}
-        />
-        <RangeDatePicker
-          startLabel='시작일'
-          endLabel='종료일'
-          getStartValue={getStartDate}
-          getEndValue={getEndDate}
-        />
-        <MbtiWrapper>
-          <MbtiHeadingWrapper>
-            <Typography variant='h5'>선호 MBTI</Typography>
-            <FormControlLabel
-              control={
-                <Checkbox
-                  value='non-preference'
-                  onChange={() => {
-                    setMbtiPreference(!mbtiPreference);
-                  }}
-                  disabled={mbtiCheckedList.length > 0 ? true : false}
+    <Formik
+      initialValues={{
+        title: '',
+        description: '',
+      }}
+      validationSchema={CreateSchema}
+      onSubmit={(values, actions) => {
+        actions.setSubmitting(true);
+        const formData = createFormData(
+          values.title,
+          values.description.trim(),
+        );
+
+        setTimeout(() => {
+          for (const key of formData.keys()) {
+            console.log(key, ':', formData.get(key));
+          }
+          actions.setSubmitting(false);
+        }, 3000);
+      }}
+    >
+      {(formik) => {
```

Comment:
p4;
`formik.handleSubmit, formik.isSubmitting` 같은 것을 빼서 사용해도 좋을 것 같습니다.
```suggestion
      {({handleSubmit, isSubmitting}) => {
```

Reply:
감사합니다👍 반영 완료했습니다! 8a85cf0cd105803b237ac0928be7d668d41ff387